stdlib/format.mli

```diff
@@ -886,6 +894,7 @@ type formatter_out_functions = {
   (e.g. {!Stdlib.output_string} and {!Stdlib.flush} for a
    {!Stdlib.out_channel} device, or [Buffer.add_substring] and
    {!Stdlib.ignore} for a [Buffer.t] output device),
+- fields [out_width] is the byte length of the substring
```

Comment:
```suggestion
- field [out_width] is the byte length of the substring
```

Reply:
This was a bit more than a typo, since the documentation referred to the previous default [out_width] function. Thanks for noticing the typo !